### PR TITLE
Skip TestHttpsInsecure test on nacl

### LINKED ISSUE #195: Fuzz TestParseData failures on nacl
I'm trying to update the vendored pprof in the Go repository, and the `nacl-386` and `nacl-amd64p32` are failing with the following error:

```
--- FAIL: TestParseData (0.37s)
	fuzz_test.go:14: Problem reading directory testdata/ : open testdata/: No such file or directory
FAIL
FAIL	cmd/vendor/github.com/google/pprof/fuzz	0.443s
2017/08/20 11:28:07 Failed: exit status 1
```

All the other trybots are fine.

The vendored pprof commit is 2191094077cac7cd05ced16c0768a2e57fe5371d.

[Full nacl-386 log](https://storage.googleapis.com/go-build-log/e0643842/nacl-386_5da3967c.log)
[Link the Go change](http://golang.org./cl/57370)

### PR DESCRIPTION
**Pull request by ALTree**
Fixes #195

### CODE REVIEW

File: fuzz/fuzz_test.go [hunk 1]
```diff
@@ -2,12 +2,17 @@ package pprof
 
 import (
 	"io/ioutil"
+	"runtime"
 	"testing"
 
 	"github.com/google/pprof/profile"
 )
 
 func TestParseData(t *testing.T) {
```
Comment (aalexand) [inline fuzz/fuzz_test.go]: The title mentions TestHttpsInsecure but TestParseData is modified?

Comment (aalexand) [inline fuzz/fuzz_test.go]: Looks like you are reusing some old branch? Skipping TestHttpsInsecure on nacl is in master for a while - https://github.com/google/pprof/blob/9277d675724626f62433309e6dd3e8ec48a3e76e/internal/driver/fetch_test.go#L355

**Comment by nolanmar511** [comment]
Closing this and opening new PR with this fix (#198)

**Comment by ALTree** [comment]
Ugh, sorry, I messed up the pull request, I never use github. Thanks for sending another.

**Pull request closed without merging**